### ISSUE TITLE
About patch size

### ISSUE THREAD

**Issue by Lnan1615** [davidiommi/Pytorch--3D-Medical-Images-Segmentation--SALMON#5]
Shouldn't the patch size obtained by volume be 16*16*16? The default for init.py is 256*256*16. When I tried to train my own model to segment the airway with CT images, the loss was consistently above 0.9. I tried to adjust the size of patch, but the loss was still not reduced. Are there any other parameters I need to modify?
